pkg/commands/options/build.go

```diff
@@ -181,7 +181,7 @@ func createBuildConfigMap(workingDirectory string, configs []build.Config) (map[
 		// the constructured import path
 		localImportPath := fmt.Sprint(".", string(filepath.Separator), path)
 
-		pkgs, err := packages.Load(&packages.Config{Mode: packages.NeedName, Dir: baseDir}, localImportPath)
+		pkgs, err := packages.Load(&packages.Config{Mode: packages.NeedName, Dir: ""}, localImportPath)
```

Comment:
We'll want to do the same dance here as everywhere else, I think.

Reply:
Done.

Reply:
I think you need to push still?

Reply:
🤦 😅 Riiiiight.